packages/devextreme/js/__internal/ui/collection/m_collection_widget.edit.ts

```diff
@@ -428,7 +434,7 @@ const CollectionWidget = BaseCollectionWidget.inherit({
     this._setAriaSelectionAttribute($itemElement, String(isSelected));
   },
 
-  _updateSelectedItems(args) {
+  updatingSelectedItems(args) {
```

Comment:
[naming]
Kindly leave the old name here because this function does exactly what is mentioned in the name - updates selected items:
-  we use `_` as a prefix for all methods which are "private" (are not supposed to be used from external)
- all functions should be named starting with a verb infinitive
```suggestion
  _updateSelectedItems(args) {
```

Reply:
[b62f308](https://github.com/DevExpress/DevExtreme/pull/27860/commits/b62f30857d83924a9f3c3b9e14c53016e7d67bdb)